_posts/2022-07-25-Building-CNF-Using-CNDP-Rust.md

```diff
@@ -0,0 +1,194 @@
+---
+layout: post
+title: Building Cloud Native Network Functions using Rust
+author: CNDP Team
+excerpt_separator: <!--more-->
+---
+
+
+## Rust for Networking and Cloud Native Development
+
+The [Rust](#rust) programming language has been steadily gaining
+popularity over the last several years. Many developers, companies use

```

Comment:
Many developers and companies, not "Many developers, companies"

Reply:
ok